include/kernels/dslash_ndeg_twisted_clover.cuh

```diff
@@ -41,7 +41,7 @@ namespace quda
     using real = typename mapper<typename Arg::Float>::type;
     using Vec = ColorSpinor<real, Arg::nColor, 4>;
     using Cache = SharedMemoryCache<Vec>;
-    using Ops = std::conditional_t<kernel_type == INTERIOR_KERNEL, KernelOps<Cache>, NoKernelOps>;
+    using Ops = KernelOps<Cache>; //std::conditional_t<kernel_type == INTERIOR_KERNEL, KernelOps<Cache>, NoKernelOps>;
```

Comment:
NOTE, this is a temporal fix because without wouldn't compile. We still have to understand why is the case.

Reply:
I think the issue is this: https://github.com/lattice/quda/pull/1447/files#diff-db133e457034e56706bf162a3cf559751de5ce8a1a62a504faa3b965395bea1aL82
Dropping the constexpr makes it always compile in the `SharedMemoryCache` which means the KernelOps must always include it.  I think the fix is to keep the `if constexpr (mykernel_type == INTERIOR_KERNEL) {` on the outside and put the `if(arg.dd_x.isZero(coord)) {...} else {...}` on the inside.